### ISSUE TITLE
Setting the keyboard variant

### ISSUE THREAD

**Issue by PaulJohnson1** [archlinux/archinstall#1042]
On [release 2.3.2](https://github.com/archlinux/archinstall/releases/tag/v2.3.2), this can be done simply by using `localectl set-x11-keymap us "" <variant>`. However, on the [current master branch](https://github.com/archlinux/archinstall/tree/ac0162aba7405f87f17cfd99c99f7d540482ee6c), that same command only works on the login screen. I have tried `setxkbmap` as well, but the layout only works on programs that are not the terminal from my testing.

I am using GDM.

**Comment by Torxed** [comment]
As a custom command?
Because we do: https://github.com/archlinux/archinstall/blob/f7aba1d31c0556b38ee48270ba530359bc274ebf/archinstall/lib/installer.py#L1052

And I was under the impression that it set the locale for X as well, or used to at least.
Could this be a bug with GDM?

**Comment by PaulJohnson1** [comment]
> As a custom command?
Yeah I just ran it when booted into the OS after I logged in through gnome.

> And I was under the impression that it set the locale for X as well, or used to at least.
The problem with not using the X version, is that there is no support for the keyboard layout I use (workman). I attempted to find out where to add the layout myself online, but have had no luck yet.

> Could this be a bug with GDM?
I don't think this is, because something between the release version and the current master branch has to have changed to cause this.
